src/test/javascript/spec/component/faq/faq.component.spec.ts

```diff
@@ -0,0 +1,183 @@
+import { HttpErrorResponse, HttpResponse } from '@angular/common/http';
+import { ComponentFixture, TestBed, fakeAsync, flush } from '@angular/core/testing';
+import { TranslateService } from '@ngx-translate/core';
+import { ActivatedRoute, Router, convertToParamMap } from '@angular/router';
+import { MockComponent, MockModule, MockProvider } from 'ng-mocks';
+import { of, throwError } from 'rxjs';
+import { MockRouterLinkDirective } from '../../helpers/mocks/directive/mock-router-link.directive';
+import { MockRouter } from '../../helpers/mocks/mock-router';
+import { MockTranslateService } from '../../helpers/mocks/service/mock-translate.service';
+import { ArtemisTestModule } from '../../test.module';
+import { FAQService } from 'app/faq/faq.service';
+import { FAQ } from 'app/entities/faq.model';
+import { ArtemisMarkdownEditorModule } from 'app/shared/markdown-editor/markdown-editor.module';
+import { MockResizeObserver } from '../../helpers/mocks/service/mock-resize-observer';
+
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FAQComponent } from 'app/faq/faq.component';
+import { FAQCategory } from 'app/entities/faq-category.model';
+import { CustomExerciseCategoryBadgeComponent } from 'app/shared/exercise-categories/custom-exercise-category-badge/custom-exercise-category-badge.component';
+import { AlertService } from 'app/core/util/alert.service';
+import { SortService } from 'app/shared/service/sort.service';
+
+function createFaq(id: number, category: string, color: string): FAQ {
+    const faq = new FAQ();
+    faq.id = id;
+    faq.questionTitle = 'questionTitle';
+    faq.questionAnswer = 'questionAnswer';
+    faq.categories = [new FAQCategory(category, color)];
+    return faq;
+}
+
+describe('FaqComponent', () => {
+    let faqComponentFixture: ComponentFixture<FAQComponent>;
+    let faqComponent: FAQComponent;
```

Comment:
I just noticed that the use of `FAQ` vs. `Faq` isn't consistent. The Server uses `Faq`, but the client generally uses `FAQ`.
Recently we had a PR that introduced the `Ide` entity and we used `Ide` everywhere, so I think it would be a good idea to use `Faq` for the model / components / services / etc.